barman/backup.py

```diff
@@ -874,7 +874,7 @@ def delete_wal(self, wal_info):
                 os.removedirs(os.path.dirname(wal_info.fullpath(self.server)))
             except OSError:
                 # This is not an error condition
-                # We always try to remove the the trailing directories,
+                # We always try to remove the trailing directories,
```

Comment:
I'm still experimenting with duplicate word detection, but I'm fairly confident about this...